calculate.js

```diff
@@ -25,4 +25,14 @@ OLD_WASTED_YEARS = OLD_WASTED_HOURS / 24 / 365
 
 document.querySelector('#life-wasted').innerText = Math.round(WASTED)
 document.querySelector('#old-life-wasted-hours').innerText = Math.round(OLD_WASTED_HOURS)
-document.querySelector('#old-life-wasted-years').innerText = Math.round(OLD_WASTED_YEARS)
\ No newline at end of file
+document.querySelector('#old-life-wasted-years').innerText = Math.round(OLD_WASTED_YEARS)
+
+//Tweet Button 
+
+function tweet(){
+  var url = 'https://twitter.com/intent/tweet';
+  var website = 'http://howmuchtimehaveiwasted.com/'
+  var text = "I've wasted " + WASTED +  " percent of my life in front of a screen."
```

Comment:
percent should be %, imo.

Reply:
I was having issues getting % to work. `%` acts as a space in URLs